aspnetcore/mvc/razor-pages/filter/sample/PageFilter/Startup2.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using PageFilter.Filters;
+
+namespace PageFilter
+{
+    public class Startup2
+    {
+        ILogger _logger;
+        public Startup2(ILoggerFactory loggerFactory, IConfiguration configuration)
+        {
+            _logger = loggerFactory.CreateLogger<GlobalFiltersLogger>();
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        #region snippet2
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc()
+               .AddRazorPagesOptions(options =>
+               {
+                   options.Conventions.AddFolderApplicationModelConvention(
+                       "/subFolder",
+                       model => model.Filters.Add(new SampleAsyncPageFilter(_logger)));
```

Comment:
Is there value in showing a sync and async filter being added globally? I'd pick one as long as we show a sample of a sync filter and an async filter being used.

Reply:
That sample is to show `AddFolderApplicationModelConvention`
In the rendered document, this async filter is shown in the previous snippet.